Makefile

```diff
@@ -258,6 +259,17 @@ ifeq ($(USE_CONTAINER),1)
 else
 	go generate ./internal/winmanifest
 endif
+
+generate-snmp:
+ifeq ($(USE_CONTAINER),1)
+	$(RERUN_IN_CONTAINER)
+else
+	@LATEST_SNMP_VERSION=$$(go list -m github.com/prometheus/snmp_exporter | cut -d' ' -f2); \
```

Comment:
Can you not get the version using [Go templates](https://dave.cheney.net/2014/09/14/go-list-your-swiss-army-knife)? It'd be more readable.

Reply:
Thanks, updated.